lib/puppet/face/certificate.rb

```diff
@@ -114,7 +114,12 @@
     EOT
 
     option("--[no-]allow-subject-alt-names") do
-      summary _("Whether or not to accept DNS alt names in the certificate request")
+      summary _("Whether or not to accept subject alt names in the certificate request")
+    end
+
+    option("--[no-]allow-dns-alt-names") do
+      summary _("Whether or not to accept subject alt names in the certificate request")
+      Puppet.deprecation_warning(_("--allow-dns-alt-names is deprecated and has been replaced by --allow-subject-alt-names"))
```

Comment:
I've just noticed some weirdness with this. I did ``be puppet help`` and this deprecation warning came up... I'm not sure what's going on but I'm going to dig into it further.